src/applications/personalization/profile360/sass/profile-360.scss

```diff
@@ -75,3 +75,14 @@ input.usa-only-phone {
 .no-wrap {
   white-space: nowrap;
 }
+
+.address-validation-container {
+  overflow: hidden;
+  position: relative;
+  width: 100%;
+  padding: 0 5px;
```

Comment:
I do wonder if this should removed and a utility class should be added to the `div` but otherwise 👍